src/tests/Dashboard/action/dictionaryAction.test.js

```diff
@@ -381,6 +382,35 @@ describe('Test suite for dictionary actions', () => {
       expect(store.getActions()).toEqual(expectedActions);
     });
   });
+
+  it('should retire/unretire a concept', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: sampleConcept,
+      });
+    });
+    const store = mockStore({ payload: {} });
+    return store.dispatch(retireConcept(sampleConcept.url, { retired: false })).then(() => {
```

Comment:
How is this testing for should unretire a concept?

Reply:
I have updated the `retireConcept()` action to return the retired concept and I am now testing directly for the retired flag. `expect(result.retired).toEqual(true);` when retiring, and `expect(result.retired).toEqual(false);` when un-retiring.